.github/workflows/ci.yml

```
@@ -13,7 +13,13 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11-dev"]
+        python-version:
+        # Build on pre-releases until stable, then stable releases.
+        # actions/setup-python#213
```

Maybe something like this?  I also expanded the link to make it easier to open.
```suggestion
        # This syntax will select the latest available releases
        # including alpha, then beta, and finally stable releases.
        # See https://github.com/actions/setup-python/issues/213
```